packages/app-project/src/screens/ClassifyPage/components/ClassifierWrapper/ClassifierWrapper.js

```diff
@@ -53,20 +54,22 @@ export default function ClassifierWrapper({
   const ownerSlug = router?.query.owner
   const projectSlug = router?.query.project
 
-  /* Only increment stats on the classify page if the subject is not retired or not already seen by current user */
-  const incrementStats = yourStats?.increment
+  /*
+    Increment user stats on every classification submitted.
+    Add the recently classified subject to the user's Recents.
+  */
+  const projectID = project?.id
+  const { mutate } = useYourProjectStats({ projectID, userID })
```

Comment:
The `mutate` function is referenced in the ClassifierWrapper component because the `useYourProjectStats()` hook must be called in a functional component. `mutate` is specific to the SWR `key` in `useYourProjectStats()`, so it can be used to update the `data` returned by the hook on every classification submitted.